src/APIs/github-def.ts

```diff
@@ -0,0 +1,7 @@
+import { Endpoints } from '@octokit/types';
+
+type GithubEndpoints = Endpoints['GET /repos/:owner/:repo/issues/:issue_number'];
```

Comment:
This should be named more explicitly incase we end up getting more information. Maybe `GHIssueEndpoint`?